db/migrations/0004_alter_moviesession_cinema_hall_and_more.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 4.0.2 on 2024-01-02 17:03
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0003_alter_movie_actors_alter_movie_genres'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='moviesession',
+            name='cinema_hall',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='movies_session', to='db.cinemahall'),
+        ),
+        migrations.AlterField(
+            model_name='moviesession',
+            name='movie',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='movies_session', to='db.movie'),
+        ),
+    ]
```

Comment:
[This comment](https://github.com/mate-academy/py-movie-moviesession-cinemahall/pull/598/files#r1439663100) is not resolved, please include one migration in this pull request.